### ISSUE TITLE
Feature Request: Option to default not include binary columns

### ISSUE THREAD

**Issue by johnwc** [HeidiSQL/HeidiSQL#1676]
**Is your feature request related to a problem? Please describe.**
When viewing a table that has large binary/spatial columns, the UI will freeze for some time while trying to either retrieve the large query or trying to load the large query in the table viewer.

**Describe the solution you'd like**
Have a option to not include binary columns in initial query of table. Like if someone already viewed the table, and unchecked the binary/spatial column.

**Describe alternatives you've considered**
We have to sit and wait until the table loads to uncheck the spatial column.

**Comment by ansgarbecker** [comment]
That sounds as if not the binary data type is the problem, but the large content, right? Note that also text columns can be large, so if at all I would not restrict such a feature to binary columns.

**Comment by johnwc** [comment]
True, but binary content is typically larger than text columns as well as not displayable. Don't really need to pull back a bunch of non-readable binary characters if it can't be read.
